db/migrate/20210403001933_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :events do |t|
+      t.string :event_name
+      t.date :event_date
+      t.text :description
+      t.references :user, foreign_key: 'creator_id'
+
+      t.timestamps
```

Comment:
Add index for `creator_id` column as in below. Remember that you do not edit your migration files by hand. You should add the index by creating a new migration.

```ruby
# Your database schema should like 

def change
  create_table :events do |t|
    t.string :event_name
    # ...rest
  end

  add_index :events, :creator_id
end
```

Reply:
Hello, Thank you a lot for your review, but I do not have a **creator_id** column  in my Event database, I do have a "user_id" it does the work of a creator_id, and it already has an index, I don't see why should I add more unnecessary column that I will not need to use it.

Please explain to me more if I'm wrong.

Reply:
I guess I was being specific on the project's specifications which say to add a `creator_id` in your `events` table.

In the code snippet below, I would like to understand what you intend to achieve with this part `foreign_key: 'creator_id'`, it seems strange, I could learn.
```ruby
t.references :user, foreign_key: 'creator_id'
```

Reply:
Hello Cyonii, good observation thanks for taking your time to reply to me :), I believe that was just me playing around and learning how to implement migrations but I later changed that code and did not use it anymore, if you look in my schema.rb the events table does not have that.